react/src/components/DoubleTag.tsx

```diff
@@ -32,7 +32,13 @@ const DoubleTag: React.FC<{
   return (
     <Flex direction="row">
       {_.map(objectValues, (objValue, idx) => {
-        return !_.isEmpty(objValue.label) ? (
+        const hasEmptyChildren =
+          React.isValidElement(objValue.label) &&
```

Comment:
We cannot determine the final rendering result using a React element. This element serves as an instruction on which component to render with which props, including children. Therefore, it would be better to refactor by changing the label type to `string` and adding a `highlightText` prop to the DoubleTag component.